code/modules/mob/living/carbon/xenomorph/castes/crusher/abilities_crusher.dm

```diff
@@ -113,32 +119,16 @@
 		big_mob_message = ", struggling mightily to heft its bulk"
 
 	if(X.a_intent == INTENT_HARM) //If we use the ability on hurt intent, we throw them in front; otherwise we throw them behind.
-		for(var/x in 1 to toss_distance)
-			temp = get_step(T, facing)
-			if (!temp)
-				break
-			T = temp
+		facing = get_dir(X, A)
 	else
 		facing = get_dir(A, X)
-		var/turf/throw_origin = get_step(T, facing)
-		if(isclosedturf(throw_origin)) //Make sure the victim can actually go to the target turf
-			to_chat(X, span_xenowarning("We try to fling [A] behind us, but there's no room!"))
-			return fail_activate()
-		if(!X.issamexenohive(A)) //xenos should be able to fling xenos into xeno passable areas!
-			for(var/obj/effect/forcefield/fog/fog in T)
-				A.balloon_alert(X, "cannot, fog")
-				return fail_activate()
-		for(var/obj/O in throw_origin)
-			if(!O.CanPass(A, get_turf(X)) && !istype(O, /obj/structure/barricade)) //Ignore barricades because they will once thrown anyway
-				to_chat(X, span_xenowarning("We try to fling [A] behind us, but there's no room!"))
-				return fail_activate()
```

Comment:
Should've really mentioned you removed this in the PR description, this is pretty huge.

Reply:
Wdym? This isn't needed anymore, there's no actual in game change. This used to be needed because crest toss would just teleport the mob BEHIND the crusher.

Reply:
Also made it so you wouldn't be able to toss with something on the tile behind you, which cucked me personally many times